tasks/skylab/StarAlign.wdl

```diff
@@ -130,3 +130,177 @@ task StarAlignFastqPairedEnd {
   }
 
 }
+
+
+task STARsoloFastq {
+  input {
+    Array[File] r1_fastq
+    Array[File] r2_fastq
+    File tar_star_reference
+    File white_list
+    String chemistry
+    String counting_mode
+
+    # runtime values
+    String docker = "quay.io/humancellatlas/secondary-analysis-star:v2.7.9a"
+    Int machine_mem_mb = 64000
```

Comment:
Does this task needs a standard memory size (64gb is pretty big) regardless of the size of the input?

Reply:
Yes, STARsolo requires a lot more memory for sorting than STAR.